samples/c2pa/C2paPlayer/C2paMenu/C2paMenuFunctions.js

```diff
@@ -98,19 +121,55 @@ export let updateC2PAMenu = function (c2paStatus, c2paMenu , isMonolithic , vide
         );
 
         if (c2paItemValue != null) {
+            //formatting for social media links
+            if(c2paItemKey=== 'SOCIAL' ){
+                var socialArray = c2paItemValue.map(function (account) {
+                    var formattedWebsite = providerInfoFromSocialId(account).name
+                    return `<span><a class="url" href="${account}" onclick="window.open('${account}')">${formattedWebsite}</a></span>`
```

Comment:
Why is `onclick` set to `window.open` if this has an `href`?

Reply:
href doesnt work and idk why :(